Makefile

```diff
@@ -33,9 +33,11 @@ TINYCBOR_SOURCES = \
 #
 CBORDUMP_SOURCES = tools/cbordump/cbordump.c
 
+BUILD_SHARED = $(shell file -L /bin/sh 2>/dev/null | grep -q ELF && echo 1)
```

Comment:
Would this override the 'make' line, or will the build system ignore this line?

Reply:
It's the other way around: the command-line overrides Makefile assignments.